robot/basestation/app.py

```diff
@@ -185,13 +187,15 @@ def initiate_feed_recording():
     if is_recording_stream(stream_url):
         return jsonify(success=False, msg="Stream is already recording")
     else:
-        success, message = start_recording_feed(stream_url);
+        success, message = start_recording_feed(stream_url)
         return jsonify(success=success, msg=message)
 
 @app.route("/stop_feed_recording/", methods=["POST", "GET"])
 def stop_feed_recording():
     stream_url = request.args['stream_url']
+    rotation = int(request.args['camera_rotation'])
     if is_recording_stream(stream_url):
+        success, message = add_rotation(stream_url, rotation)
         success, message = stop_recording_feed(stream_url)
```

Comment:
```suggestion
        success, message = add_rotation(stream_url, rotation)
        
        if not success:
            print('add_rotation method failed:', message)
            
        success, message = stop_recording_feed(stream_url)
        
        if not success:
            print('stop_recording_feed method failed:', message)
```

Reply:
let's make use of the potential error messages here, otherwise we're just overwriting those variables without using them at all (aside from sending the final message back to GUI)

Reply:
True, I am adding that